src/pages/docs/endpoints/examples.mdx

```diff
@@ -79,6 +79,51 @@ A single example JSON object will be returned.
   "style": "",
   "associatedDefinitionsSchemas": [],
 }
+ ```
+###### Proverbs
+ Proverbs are gotten with the `examples` endpoint the same way examples are gotten. The only exception
+ is that the `style` parameter is set to `proverb`.
+
+ For example, this is a GET request for three (3) proverbs about `war`.
+
+ **Parameters**
+
+| Name   | Description | Example |
+| :----- | :----- | :----- |
+| **keyword** *required* | Keyword for querying words |  `keyword=war` |
+| **style** | Style for requested example sentences | `style=proverb` |
+| **page** | Page for results | `page=1` |
+| **range** | Page for result using [x,y] syntax, max range includes 25 documents | `range=[0, 2]` |
```

Comment:
I feel like this section is a direct copy of the the "###### Get a Single Example" section but the only difference is that you provide the `style=proverb` field

Maybe we should consider adding a subsection in the "###### Get a Single Example" section that talks about proverbs?

Reply:
I just did that. I'll push so you can review it.

Reply:
I think that should be a better option. Though since my example had 3 examples, I placed it under the '###### Get Multiple Examples' section. 

I think the single example can be understood. Or do I need to add that too?

Reply:
@chidiadi01 this is moving in the right direction! but we are still duplicating a lot of text.

there's a typo that needs to be fixed to make this PR mergeable.

screenshot of the typo: 
![image](https://github.com/nkowaokwu/igbo_api/assets/16169291/c6d78e33-6cf2-4a45-9086-da039fd6108f)

you can see that we list all valid values of what can be provided in the `style` parameter, but those values don't appear in the rendered markdown.

rather than adding the next lines you have included in this PR, you should include those extra parameter values inside the backticks (i.e. `style=no_stye|standard|proverb|biblical`)